codec-http/src/main/java/io/netty/handler/codec/http/multipart/DiskFileUpload.java

```diff
@@ -157,8 +160,7 @@ protected String getBaseDirectory() {
 
     @Override
     protected String getDiskFilename() {
-        File file = new File(filename);
-        return file.getName();
+        return String.valueOf(fileNameCounter.incrementAndGet());
```

Comment:
That's what I meant with "can mark the file for deletion when the JVM exits", I have it before thats why I'm mentioning this.